docs/configuration.md

```diff
@@ -25,6 +25,20 @@ export default defineConfig({
 });
 ```
 
+## `endpoint`
+
+The endpoint to use for the Happo run. Defaults to `https://happo.io`. This is
+used for on-premise Happo.
```

Comment:
"This" here is ambiguous, it could refer to the option or the default value. We should clarify (maybe by combining it with the first sentence?)

Reply:
I'm also going to move it toward the end of the doc, since most people won't need it.